jupyterlab_gitsync/src/components/file.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { fileEntry } from '../style/files';
+
+export interface Props {
+  path: string;
+}
+
+export class File extends React.Component<Props> {
+  /**
+   * Returns a React component for rendering a panel toolbar.
+   *
+   * @param props - component properties
+   * @returns React component
+   */
+
+  constructor(props: Props) {
+    super(props);
+  }
+
+  /**
+   * Renders the component.
+   *
+   * @returns React element
+   */
+  render(): React.ReactElement {
+    return (
+      <tr className={fileEntry}>
+        <td>{this.props.path}</td>
+        <td>
+          <button type="button" onClick={this._onResolveClick}>
+            {' '}
```

Comment:
Are these extra space characters necessary here around the button text?

Reply:
This is just a placeholder file right now, but I'll add a TODO comment so it's more clear.